R/keypair.R

```diff
@@ -0,0 +1,92 @@
+# Support for Snowflake key-pair authentication for SPCS and the REST API.
+keypair_credentials <- function(
+  account,
```

Comment:
Consider using `snowflake_` prefixes for some of these argument names to remind folks that this is not the rsconnect account name used elsewhere.